plugins/colored-man/colored-man.plugin.zsh

```diff
@@ -19,12 +19,13 @@ fi
 man() {
       env \
       	  LESS_TERMCAP_mb=$(printf "\e[1;31m") \
-	  LESS_TERMCAP_md=$(printf "\e[1;31m") \
+	  LESS_TERMCAP_mb=$(printf "\e[1;31m") \
```

Comment:
Looks like this is redundantly defining LESS_TERMCAP_mb, which is also set on line 21.